dpgen2/op/run_lmp.py

```diff
@@ -65,6 +87,52 @@ def execute(
         TransientError
             On the failure of LAMMPS execution. Handle different failure cases? e.g. loss atoms.
         """
-        raise NotImplementedError
+        config = ip['config'] if ip['config'] is not None else {}
+        config = RunLmp.normalize_config(config)
+        command = config['command']
+        task_name = ip['task_name']
+        task_path = ip['task_path']
+        models = ip['models']
+        input_files = [lmp_conf_name, lmp_input_name]
+        input_files = [(Path(task_path)/ii).resolve() for ii in input_files]
+        model_files = [Path(ii).resolve() for ii in models]
+        work_dir = Path(task_name)
+
+        with set_directory(work_dir):
+            # link input files
+            for ii in input_files:
+                iname = ii.name
+                Path(iname).symlink_to(ii)
+            # link models
+            for mm in model_files:
+                mname = mm.name
+                Path(mname).symlink_to(mm)
+            # run lmp
+            command = [command, '-i', lmp_input_name]
```

Comment:
-log lmp_log_name

I suggest to assign it. See https://docs.lammps.org/Run_options.html#log.